.github/workflows/deploy.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
           fetch-depth: 0
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.16.5
+        uses: pypa/cibuildwheel@v2.19.1
```

Comment:
These are intentionally pinned for stability... will need to review, blocking over this.